content/pt/docs/concepts/instrumentation/zero-code.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Autoinstrumentação
```

Comment:
Em inglês o termo `auto-instrumentation` foi substituído por `zero-code`, acho que vale a pena fazer uma substituição similar aqui pra manter consistência
Talvez `Sem código` ?

Conforme for decidido, várias ocorrências do mesmo termo nessa página terão que ser atualizadas